tldr.py

```diff
@@ -266,10 +272,15 @@ def main():
                         choices=['linux', 'osx', 'sunos'],
                         help="Override the operating system [linux, osx, sunos]")
 
+    parser.add_argument('-s', '--source',
+                        default="http://raw.github.com/tldr-pages/tldr/master/pages",
```

Comment:
Use the `DEFAULT_REMOTE` variable here ?

Reply:
Ok